build.gradle

```diff
@@ -70,23 +74,18 @@ subprojects {
     compileKotlin {
         kotlinOptions {
             freeCompilerArgs = ["-Xjsr305=strict"]
-            jvmTarget = 1.8
         }
     }
 
     compileTestKotlin {
         kotlinOptions {
             freeCompilerArgs = ["-Xjsr305=strict"]
-            jvmTarget = 1.8
         }
     }
 
     kotlinter {
         ignoreFailures = false
-        indentSize = 4
```

Comment:
explain please @mversic

Reply:
the parameter is no longer supported by kotlinter